### ISSUE TITLE
All log output is responding with invalid value

### ISSUE THREAD

**Issue by Timer** [lensapp/lens#4]
```
invalid value for "since": parsing time "-62135596800" as "2006-01-02": cannot parse "-62135596800" as "2006"
```

![image](https://user-images.githubusercontent.com/616428/49377668-c697b500-f6d8-11e8-9ae8-49e6fbadda2e.png)

```
HTTP/1.1 200 OK
Audit-Id: 89f65505-9cfe-4f30-b4b8-a8078546432a
Content-Length: 109
Content-Type: text/plain
Date: Mon, 03 Dec 2018 13:52:53 GMT
X-Powered-By: Express

invalid value for "since": parsing time "-62135596800" as "2006-01-02": cannot parse "-62135596800" as "2006"
```

---

Request has empty since time:
```
curl "https://lens.int.foundationsoft.com/api-kube/api/v1/namespaces/registry/pods/stolon-sentinel-76d8c58b74-hz6x2/log?container=stolon-sentinel^&timestamps=true^&tailLines=1000^&sinceTime="
```

--- 

This is lens 1.3.0 shipped with Pharos `2.1.0-beta.1`.

**Comment by nevalla** [comment]
Will be fixed in `1.3.1`

**Comment by Timer** [comment]
I'll close this then. Thanks!
